engine/server/sv_client.c

```diff
@@ -3150,7 +3092,8 @@ void SV_ConnectionlessPacket( netadr_t from, sizebuf_t *msg )
 	pcmd = Cmd_Argv( 0 );
 	Con_Reportf( "SV_ConnectionlessPacket: %s : %s\n", NET_AdrToString( from ), pcmd );
 
-	if( !Q_strcmp( pcmd, "ping" )) SV_Ping( from );
+	if (SV_SourceQuery_HandleConnnectionlessPacket ( pcmd, from ) ) return;
```

Comment:
IMHO it should first check our packets, and then check Source ones.

Source packets have single letter header, and it clashes with our packets. Like `i` vs `info`.

Reply:
Agree about ``info`` but...

goldsrc-style A2A_PING really used anywhere now inside xash ? Probably it should be dropped in favor of actual goldsrc (source-style) query protocol ?

Reply:
These packets are still the part of the protocol.

A2A_PING would be nice to use in the next ServerBrowser iteration with accurate ping times, detailed server info and other, but then I would use Xash's own protocol anyway.

Reply:
okay, returned back old-style A2A_PING, source-query now doesn't handle these type of packet.